src/core/component.ts

```diff
@@ -232,6 +232,22 @@ const BaseComponent: VueConstructor<IBaseComponent> = Vue.extend({
     }
 });
 
+function getElement(createElement, children, id) {
+    return createElement("div", {
+        attrs: { id }
+    }, children);
+}
+
+function getConfigurationNodes(nodes: VNode[]) {
```

Comment:
How about merging `getConfigurationNodes` and `getNodes` into one, smth like  `splitNodes`?
```
function splitNodes(nodes) : {  domNodes, configNodes } {
  for(var node of nodes) {
    if(node.componentOptions && .componentOptions as any as IExtensionComponentNode).$_hasOwner)
      configNodes.push(node)
    else
      domNodes.push(node)
  }
}
```

As for me it looks more clear and guarantees that no node can be lost (since single `if` statement used)